Common/Code.ts

```diff
@@ -204,13 +204,20 @@ function getStyle(equationStringEncoded: string, quality: number, renderer: Rend
   const equation: string[] = [];
   equationStringEncoded = equationStringEncoded;
   reportDeltaTime(307);
+  // handle RGB coloring, except on Texrendr
   if (renderer[5] !== "Texrendr") {
-    if (isInline) {
-      equationStringEncoded = renderer[3] + "%7B%5Ccolor%5BRGB%5D%7B" + red + "%2C" + green + "%2C" + blue + "%7D" + equationStringEncoded + renderer[4] + "%7D";
-    } else {
-      equationStringEncoded = "%7B%5Ccolor%5BRGB%5D%7B" + red + "%2C" + green + "%2C" + blue + "%7D" + equationStringEncoded + "%7D";
-    }
+    // \color[RGB]{0,0,0}
+    equationStringEncoded = "%5Ccolor%5BRGB%5D%7B" + red + "%2C" + green + "%2C" + blue + "%7D" + equationStringEncoded;
+  }
+  
+  if (isInline) {
+    // wrap in renderer inline delimiters
+    equationStringEncoded = renderer[3] + equationStringEncoded + renderer[4];
   }
+
+  // wrap in curly braces
+  equationStringEncoded = "%7B" + equationStringEncoded + "%7D";
```

Comment:
In the original code, this and the isInline condition have a switched order. Does it matter?

Reply:
You're right; that definitely does matter.
I'm pretty sure it's actually supposed to be something like `\inline { equation }`. I don't think the curly braces are needed when it's not inline, but I'll leave them in because that's how it was before.

Reply:
The updated code results in `\inline {\color[RGB]{255,0,0}3x^2}` for inline and `{\color[RGB]{255,0,0}3x^2}` for display